### PR TITLE
Added tryHalogen backend

### PR DESCRIPTION
**Pull request by sudhirvkumar**
@paf31 Sending you the PR as discussed in IRC

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -287,23 +275,29 @@ $(function() {
 
     var scripts = [bundle, wrapped].join("\n");
 
-    var script = iframe.createElement('script');
```
Comment (paf31) [inline js/index.js]: Did you test this on other backends? I seem to remember it was important.

Comment (sudhirvkumar) [inline js/index.js]: I checked every single backend available in try.purescript.org just now... (twice) and everything is working... including Halogen

Comment (sudhirvkumar) [inline js/index.js]: Having said that I tested only in Chrome!

**Comment by paf31** [comment]
Could you please rebase this? I've added a `DEPENDENCIES` file which you could regenerate in order to give attribution here.

Thanks!

**Comment by sudhirvkumar** [comment]
@paf31 sure will do

**Comment by RAbraham** [comment]
Thanks @sudhirvkumar for working on this. Excited to play with it.

**Comment by rintcius** [comment]
@sudhirvkumar let me know if you don't have time to finish this. I'm happy to help

**Comment by hdgarrood** [comment]
Thanks for the PR, although I'm going to close it as we are now using a different design where different backends are no longer a thing, see #128. Getting Halogen examples working with the new design will now hopefully just be a case of updating the `psc-package.json` file to include it.

**Pull request closed without merging**